packages/ui/components/ui/tx/view/swap.tsx

```diff
@@ -3,17 +3,10 @@ import { SwapView } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/
 import { fromBaseUnitAmount, joinLoHiAmount } from '@penumbra-zone/types/src/amount';
 import { uint8ArrayToBase64 } from '@penumbra-zone/types/src/base64';
 import { ActionDetails } from './action-details';
-import { AddressView } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/keys/v1/keys_pb';
-import { AddressViewComponent } from './address-view';
 
 export const SwapViewComponent = ({ value }: { value: SwapView }) => {
```

Comment:
We no longer need to show the claim address. Here's what that looks like in the approval dialog (it's the same in the Swap transaction view of a successful transaction, too):

<img width="512" alt="image" src="https://github.com/penumbra-zone/web/assets/1121544/a85aec56-93e5-459a-9757-d0fbb676157c">

Reply:
Hmm. I'm not sure we should hide it. The claiming part of a swap seems so integral to penumbra. I don't think it hurts to show it.

Reply:
Hm OK. I was basing it on Henry's comment in #548:

> Adding this check means that we don't need to display the claim address to the user in the approval view.

That said, I think the main point of that ticket was to ensure the claim address belonged to the current user, so I'll add it back.

Reply:
Then up to you! 🤷

Reply:
Yeah, I'd prefer to hide it. The user shouldn't ever have to know about the claim address, it should just do the right thing automatically.

Reply:
As a follow-up to this PR, we should also change the swap rendering to show formatted values, so that as a user I see something like "Swap 1234 UM => TestUsd"